### PR TITLE
added status routes

### PR DESCRIPTION
**Pull request by fredmaggiowski**

### CODE REVIEW

File: src/main/java/eu/miaplatform/customplugin/springboot/StatusController.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ */
+
+package eu.miaplatform.customplugin.springboot;
+
+import io.swagger.annotations.Api;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@Api(value = "Status Routes")
```
Comment (davidebianchi) [inline src/main/java/eu/miaplatform/customplugin/springboot/StatusController.java]: How it is used this value?

File: src/test/java/eu/miaplatform/customplugin/springboot/StatusControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+public class StatusControllerTest {
+    @Autowired
+    private MockMvc mvc;
+
+    @Test
+    public void defaultReadinessRoute() throws Exception {
+        mvc.perform(MockMvcRequestBuilders.get("/-/ready"))
+                .andExpect(status().isOk())
+                .andExpect(content().json(
+                        "{\"name\":\"Custom Plugin Spring Boot REST API\",\"version\":\"0.0.1-SNAPSHOT\",\"status\":\"OK\"}"
```
Comment (davidebianchi) [inline src/test/java/eu/miaplatform/customplugin/springboot/StatusControllerTest.java]: Those tests will break when, once created the service through the console, version will be changed. Could you change this tests to not break after version upgrade?

**Review by davidebianchi — CHANGES_REQUESTED** [review]

File: src/test/java/eu/miaplatform/customplugin/springboot/StatusControllerTest.java (hunk 2 above)
Comment (fredmaggiowski) [reply src/test/java/eu/miaplatform/customplugin/springboot/StatusControllerTest.java]: ok, I've improved them

File: src/main/java/eu/miaplatform/customplugin/springboot/StatusController.java (hunk 1 above)
Comment (fredmaggiowski) [reply src/main/java/eu/miaplatform/customplugin/springboot/StatusController.java]: It's used by springfox but is documented as not working / deprecated

I've removed it

**Review by davidebianchi — APPROVED** [review]

**Pull request merged**